endpoints/openrtb2/video_auction.go

```diff
@@ -260,6 +260,8 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 	// all code after this line should use the bidReqWrapper instead of bidReq directly
 	bidReqWrapper := &openrtb_ext.RequestWrapper{BidRequest: bidReq}
 
+	openrtb_ext.ConvertUpTo26(bidReqWrapper)
```

Comment:
There are a few fields being set on the request like `id` and `test` prior to the upconvert but that should be ok.

Reply:
Moving the conversion to the wrapper up would be a lot of work to make everything wrapper compatible. So as you said, it should work well here.